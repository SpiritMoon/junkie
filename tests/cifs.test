#!/bin/sh

NAME="CIFS"
PCAPS="pcap/smb/smb.pcap pcap/smb/splitted_smb.pcap"
filter() {
    grep -E '^(Netbios|CIFS)'
}
OUTPUT="cifs.out"

. "$srcdir/dump_test"
