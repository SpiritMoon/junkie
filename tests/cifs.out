-- pcap/smb/smb.pcap --
Netbios: head_len=4, payload=70
CIFS: head_len=32, payload=38, command=SMB_COM_TRANSACTION2, status=SMB_STATUS_OK, trans2_subcmd=SMB_TRANS2_QUERY_FS_INFO, query, query_write_bytes=0, response_read_bytes=0, response_write_bytes=0, meta_read_bytes=0, meta_write_bytes=0
Netbios: head_len=4, payload=35
CIFS: head_len=32, payload=3, command=SMB_COM_TRANSACTION2, status=SMB_STATUS_OS2_INVALID_LEVEL, query_write_bytes=0, response_read_bytes=0, response_write_bytes=0, meta_read_bytes=0, meta_write_bytes=0
Netbios: head_len=4, payload=84
CIFS: head_len=32, payload=52, command=SMB_COM_DELETE, status=SMB_STATUS_OK, path=\torture_qfileinfo.txt, query, query_write_bytes=0, response_read_bytes=0, response_write_bytes=0, meta_read_bytes=0, meta_write_bytes=0
Netbios: head_len=4, payload=35
CIFS: head_len=32, payload=3, command=SMB_COM_DELETE, status=SMB_STATUS_OBJECT_NAME_NOT_FOUND, query_write_bytes=0, response_read_bytes=0, response_write_bytes=0, meta_read_bytes=0, meta_write_bytes=0
